test_communication/CMakeLists.txt

```diff
@@ -322,5 +323,7 @@ if(BUILD_TESTING)
   call_for_each_rmw_implementation(targets)
 endif()  # BUILD_TESTING
 
-# TODO should not install anything
+install(FILES test/__init__.py test/message_fixtures.py
+  DESTINATION "${PYTHON_INSTALL_DIR}/${PROJECT_NAME}")
```

Comment:
they were not needed by https://github.com/ros2/cli_tools/pull/5 that motivated this PR in the first place. Fixtures for all languages will be moved in (and installed by) the package that will be created in https://github.com/ros2/system_tests/issues/191